lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -170,16 +171,24 @@ class EngineAutofillForm {
     keys.forEach((String key) {
       final html.Element element = elements![key]!;
       subscriptions.add(element.onInput.listen((html.Event e) {
-        _handleChange(element, key);
+        if (items![key] == null) {
+          throw StateError(
+              'Autofill would not work withuot Autofill value set');
+        } else {
+          final AutofillInfo autofillInfo = items![key] as AutofillInfo;
```

Comment:
Is there a better way of handling non-nullability here? @yjbanov

Reply:
Removed this part. 

Thanks for the suggestion, it was also giving error Autofill? cannot be passed to Autofill when I tried to use a local variable.

Reply:
There are two things that can be null, `items` and `items![key]`.

I imagine if we're null-asserting `items`, then we should make the field itself non-null.

Otherwise, @mdebbar's suggestion should work.